test/stdlib/UnsafeRawBufferPointer.swift

```diff
@@ -421,6 +421,21 @@ UnsafeRawBufferPointerTestSuite.test("copyMemory.overflow") {
       from: UnsafeRawBufferPointer(buffer))
 }
 
+// Use copyBytes without contiguouseStorage
+UnsafeRawBufferPointerTestSuite.test("copyBytes.withoutContiguouseStorage") {
+  let array: [UInt8] = [0, 1, 2, 3]
+  var array2 = [UInt8](repeating: 0, count: 1)
+  let ranges = [0..<1, 1..<2, 2..<3, 3..<4]
+  for range in ranges {
+    let slice = Array(array[range])
+    array2.withUnsafeMutableBytes { byte in
+        byte.copyBytes(from: slice)
+    }
+    expectEqual(array2.count, slice.count)
+    expectEqual(array2, slice)
```

Comment:
Here's a tweak:
```suggestion
var array = [UInt8](repeating: 0, count: 2)
let ranges: [Range<UInt8>] = [0..<2, 1..<3, 2..<4, 3..<5]
for range in ranges {
  array.withUnsafeMutableBytes { byte in
    byte.copyBytes(from: range)
  }
  expectEqual(array.count, range.count)
  expectEqual(array, Array(range))
}
```
I haven't verified yet that it goes through the iteration loop below the `wCSIA` invocation in `copyBytes`, but I think it would.